givitsite/givitsite/settings.py

```diff
@@ -113,7 +116,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Asia/Jerusalem'
```

Comment:
No mention of this change in the PR message...

I'd generally expect servers ti use UTC internally and just display the right TZ according to where the user is, but since your app is just Israelis you probably can get away with this.

Reply:
the problem is that i do some of the search acording to the same time so i need it, i plain to update the readme.md with explanation about it to make it clear.

Reply:
Ok. I see you're using `datetime.now()` in `find_new_furniture`. I would solve the issue there and not here - make sure there to explicitly ask for the time in the right zine, don't just assume there that the server as a whole happens to be set to Israel time...

Also I wouldn't use `now()` as a relation point in any case - instead relate to some other event - newer then the last time we've searched, newer then when the user placed the request, etc. Also when you compare times - always convert then to a standard universal format such as UTC or, even better, seconds since the UNIX epoch - otherwise you'll start having strage issues with daylight savings changes, leap years, etc.

Reply:
ok, i will fix it in the future, i open issue for all the timezone stuff -  #72